src/keyword.rs

```diff
@@ -33,7 +33,7 @@ impl Keyword {
     pub fn find_by_keyword(conn: &GenericConnection, name: &str)
                            -> CargoResult<Option<Keyword>> {
         let stmt = try!(conn.prepare("SELECT * FROM keywords \
-                                      WHERE keyword = $1"));
+                                      WHERE LOWER(keyword) = $1"));
```

Comment:
This might not hit the index because it's strictly on just the `keyword` column, but I think here we'll actually want `LOWER($1)` instead as we want to lowercase the user input